### DISCUSSION TITLE
Welcome to Adventure-Colussus Discussions!

### DISCUSSION THREAD

**Discussion by Slow-Scallion4183** [Slow-Scallion4183/Adventure-Colussus#1] [General]
## 👋 Hello!
  We’re using Discussions as a place to connect with members of our community when working on Adventure Colussus. 

Some Github guidelines:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. 
  * Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community + maybe some ideas! Happy coding :D

**Comment by wireboy5** [comment]
Hello @Slow-Scallion4183, you may know me from Reddit as u/rileyjwilton.

I am not a beginner programmer and am mostly in the realm of advanced programming, so if you need any help feel free to ask.

Because you said on Reddit you are a beginner, It immediately caught my attention because I did not have much help starting programming. I think that I can provide some tips, tricks, and ways to tidy up your code. All in all, you look to be doing a good job for the first version of a text-adventure. Because I have a busy schedule, I will be popping in here when I get the chance to help out, but will sadly not be able to reliably work on the project. Still, I hope that the assistance I will give can be worth something and will help you along your programming journey. Happy Coding!

**Reply by Slow-Scallion4183** [reply]
Amazing! Its great that you stopped by! Any assistance would be greatly appreciated especially from a seasoned and respectable programmer! Happy Coding to you to to! Good luck :D

**Comment by robotlightsyou** [comment]
Hi, I've been studying python casually for a couple years after work, but didn't get serious about it until the pandemic. I've built a bunch a basic things(tictactoe, hangman, and similar), but mostly script sized stuff, not real programs or applications. I've also never contributed to someone else's project, thank you for the opportunity to get this experience. I'm particularly interested in working on the backend of this. While I'm happy to contribute to the story and design, making the moving pieces move is what gets me going. Howdy all! (yes, I'm a dork.)
